docs/api/websocket_json_rpc_api.md

```diff
@@ -6,32 +6,35 @@ The Lisk Service is a web application that interacts with the whole Lisk ecosyst
 
 ## Table of Contents
 
-   * [Lisk Service WebSocket JSON-RPC API Documentation](#lisk-service-websocket-json-rpc-api-documentation)
-      * [Access paths and compatibility](#access-paths-and-compatibility)
-      * [Endpoint Logic](#endpoint-logic)
-      * [Responses](#responses)
-      * [Date Format](#date-format)
-      * [Multi-Requests](#multi-requests)
-   * [Lisk Blockchain-related Endpoints](#lisk-blockchain-related-endpoints)
-      * [Accounts](#accounts)
-         * [get.accounts](#getaccounts)
-         * [get.votes](#getvotes)
-         * [get.voters](#getvoters)
-      * [Blocks](#blocks)
-         * [get.blocks](#getblocks)
-      * [Delegates](#delegates)
-         * [get.delegates](#getdelegates)
-         * [get.delegates.next_forgers](#getnext_forgers)
-      * [Peers](#peers)
-         * [get.peers](#getpeers)
-      * [Transactions](#transactions)
-         * [get.transactions](#gettransactions)
-         * [get.transactions.statistics.day](#gettransactionsstatisticsday)
-         * [get.transactions.statistics.month](#gettransactionsstatisticsmonth)
-      * [Network](#network)
-         * [get.network.status](#getnetworkstatus)
-         * [get.network.statistics](#getnetworkstatistics)
-         * [get.search](#getsearch)
+<!-- TOC depthto:3 orderedlist:true -->
+
+- [1.1. Table of Contents](#11-table-of-contents)
+- [1.2. Access paths and compatibility](#12-access-paths-and-compatibility)
+- [1.3. Endpoint Logic](#13-endpoint-logic)
+- [1.4. Responses](#14-responses)
+- [1.5. Date Format](#15-date-format)
+- [1.6. Multi-Requests](#16-multi-requests)
+- [2.1. Accounts](#21-accounts)
+	- [2.1.1. get.accounts](#211-getaccounts)
+	- [2.1.2. get.votes_sent](#212-getvotes_sent)
+	- [2.1.3. get.votes_received](#213-getvotes_received)
+- [2.2. Blocks](#22-blocks)
+	- [2.2.1. get.blocks](#221-getblocks)
+- [2.3. Delegates](#23-delegates)
+	- [2.3.1. get.delegates](#231-getdelegates)
+	- [2.3.2. get.delegates.next_forgers](#232-getdelegatesnext_forgers)
+- [2.4. Peers](#24-peers)
+	- [2.4.1. get.peers](#241-getpeers)
+- [2.5. Transactions](#25-transactions)
+	- [2.5.1. get.transactions](#251-gettransactions)
+	- [2.5.2. get.transactions.statistics.day](#252-gettransactionsstatisticsday)
+	- [2.5.3. get.transactions.statistics.month](#253-gettransactionsstatisticsmonth)
+- [2.6. Network](#26-network)
+	- [2.6.1. get.network.status](#261-getnetworkstatus)
+	- [2.6.2. get.network.statistics](#262-getnetworkstatistics)
+	- [2.6.3. get.search](#263-getsearch)
+
+<!-- /TOC -->
```

Comment:
Just a remark: If you would write your Readme files in assciidoc, you can autogenerate those table of contents by adding a simple `:toc:` attriubte to the page :)

Reply:
Thanks Mona, does it help you when I add the following code block?
```
<!-- AsciiDoc TOC marker
:toc:
-->
```

Reply:
No, there is no help for me, it was just a recommendation that would make your life easier writing READMEs ;)

Reply:
I need them for Markdown anyway. But thanks for the recommendation.